docs/modules/commons-operator/pages/reference/index.adoc

```diff
@@ -0,0 +1,9 @@
+= Reference
+
+Consult the reference documentation section to find exhaustive information on:
+
+* Descriptions and default values of all properties in the CRDs used by this operator in the xref:reference/crds.adoc[].
+
+== Command line parameters and environment variables
+
+At the moment this operator accepts no command line parameters and does not read any environment variables.
```

Comment:
I mean it has the usual
```
        Command::Run(ProductOperatorRun {
            product_config: _,
            watch_namespace: _,
            tracing_target,
        }) =>
```
But ignores the watch namespace :/
Created https://github.com/stackabletech/commons-operator/pull/193
Assuming we merge the PR we should document `--watch-namespace`, especially as commons-op consumes the most memory of all operators

Reply:
Yeah this was more of a refactoring, and it was already wrong. Also, we should document this in an entirely different way, i.e. explain how to do it in Helm and OpenShift, because that's what users care about. I'd like to merge this as-is and then think about how to refactor this bit for all operators in general

Reply:
Alright